### DISCUSSION TITLE
Synchronisierung zwischen Handys

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#475] [Q&A] [answered]
Ist das korrekt ? Wenn ich bei meinem Privathandy Routen erstelle und unter 
" Routen " speicher dann erscheint diese Route nicht unter " Meine Orte - Routen " auf meinen " Navi Handy " 
Das heißt ich muss mir die Route per Mail schicken und dann auf dem " Navi Handy" abspeichern ?

**Comment by devemux86** [comment, marked as answer]
Cruiser stores all your data offline and privately.
In the next version you can share routes, favorites, profiles with other devices, even without internet.

You can use a cloud service such as Google Drive, Dropbox, etc.
We could also include some popular cloud services in the future.

- https://github.com/devemux86/cruiser/discussions/330#discussioncomment-8782827

---

Cruiser speichert alle Ihre Daten offline und privat.
In der nächsten Version können Sie Routen, Favoriten, Profile mit anderen Geräten teilen, auch ohne Internet.

Sie können einen Cloud-Dienst wie Google Drive, Dropbox, etc. verwenden.
Wir könnten in Zukunft auch einige beliebte Cloud-Dienste einbeziehen.

- https://github.com/devemux86/cruiser/discussions/330#discussioncomment-8782827

**Reply by 1maxi2** [reply]
O.k. Danke. Google drive verwende ich inzwischen um von allen Geräten auf die Routen zugreifen zu können.
